### PR TITLE
100% width for form items

### PR DESCRIPTION
**Pull request by mechanoid23**

### CODE REVIEW

File: src/scss/_osu_layout_builder.scss [hunk 1]
```diff
@@ -23,8 +30,10 @@
     background-image: url(../assets/fade-zoom-in.svg);
   }
 
-  div.form-item {
-    width: initial !important;
+  .form-item {
+    /* Replace `initial` - DRUP8-1056 */
+    width: 100%;
+    //width: initial !important;
```
Comment (Ruefa) [inline src/scss/_osu_layout_builder.scss]: Do we need to keep this comment?

**Review by Ruefa — CHANGES_REQUESTED** [review]

File: src/scss/_osu_layout_builder.scss (hunk 1 above)
Comment (mechanoid23) [reply src/scss/_osu_layout_builder.scss]: Thanks for catching that!

**Comment by emerham** [comment]
Can we also add a fix specifically to address the radios?
Screenshot from 2022-07-18 16-57-27

```css
#drupal-off-canvas .form-radios > div,
#layout-builder-modal .form-radios > div {
  width: initial;
}
```

Screenshot from 2022-07-18 17-04-16

**Review by emerham — CHANGES_REQUESTED** [review]
comment above about radios.

**Review by Ruefa — APPROVED** [review]

**Review by emerham — APPROVED** [review]
LGTM

**Pull request merged**